Resources/Prototypes/SS220/Shitspawn/Ebent/CustomItems.yml

```
@@ -42,7 +43,7 @@
 - type: entity
   parent: ClothingBeltSyndieHolster
   id: ClothingBeltSyndieHolsterPython
-  suffix: Shitspawn
+  suffix: Shitspawn, Ebent
```

```suggestion
  suffix: Shitspawn, Ebent, Python
```

Потому что это впринципе юзабельный предмет в щитспавне, но без его спавна не понять, что внутри, так как Айди в меню не показывается